### DISCUSSION TITLE
Νέες ώρες παρουσίασης για την 24 Νοεμβρίου?

### DISCUSSION THREAD

**Discussion by p14kairas** [courses-ionio/hci#1636] [Announcements]
Θα ήθελα να μου πείτε εάν θα υπάρξουν νέες ώρες για την 24 Νοεμβρίου ή αν ολόκληρη η ημέρα είναι κλεισμενη και απομένουν μόνο Δευτερα 22 και Τρίτη 23.

**Comment by epidrome** [comment]
Αν η Τετ 24 Νοε σε βολεύει για το ταξίδι σου έλα στις 4μμ, 

έχω μια άλλη συνάντηση στο ημερολόγιο 4μμ και το κάλεντλι μπλοκάρει τις ώρες 4-5.30, επίσης είναι διαθέσιμες οι ώρες 5.30-6μμ

**Reply by p14kairas** [reply]
Eντάξει 4μμ την Τετ 24 Νοε με βολεύει σημειώστε με αν θέλετε. @epidrome
